File: python/Rental_plik.py
from Car_plik import Car            #odwolanie
from Customer_plik import Customer  #odwolanie

class Rental:
    def __init__(self, customer: Customer, samochod: Car, dni: int):
        self.customer = customer
        self.samochod = samochod
        self.dni = dni

    def wyliczenie_kosztu(self):                # oblicza koszta za pomocą Dni z tego pliku * Cena z klasy Car
        return self.dni * self.samochod.cena    
    
    def wyswietl_szczegoly_wypozyczenia(self):
        print("Szczegóły Wypozyczenia")
        self.customer.klient()          #odnosi się do customer - Customer, a w nim do klient
        self.samochod.wyswietl_info()   #odnosi sie do samochod - Car, a w nim do wyswietl_info
        print(f"Liczba dni: {self.dni}")
        print(f"Koszt całkowity: {self.wyliczenie_kosztu():.2f} zł")
        print("----")
        

File: c#/wypozyczalnia_samochodow_main/Rental_plik.cs
namespace Rental_plik;   // by zainicjowac w Program.cs

using Car_plik;          // odwolanie do pliku Car_plik.cs
using Customer_plik;    // odwolanie do pliku Customer_plik.cs

public class Rental
{
    public Customer Customer { get; }    // deklaracja Customer z jego danymi typu imie, nazwisko, mail
    public Car Car { get; }             // deklaracja Car z jego danymi typu marka, model, typ, cena
    public int Dni { get; }


    public Rental(Customer customer, Car car, int dni)
    {
        Customer = customer;
        Car = car;
        Dni = dni;
    }

    public double wyliczenie_kosztu()       // oblicza koszta za pomocą Dni z tego pliku * Cena z klasy Car
    {
        return Math.Round(Dni * Car.Cena, 2);
    }

    public void wyswietl_szczegoly_wypozyczenia()       // wyswietla info o wypozyczeniach
    {
        Console.WriteLine("Szczegóły Wypozyczenia");
        Customer.klient();
        Car.wyswietl_info();
        Console.WriteLine($"Liczba dni: {Dni}");
        Console.WriteLine($"Koszt całkowity: {wyliczenie_kosztu()} zł");
        Console.WriteLine("----");
    }
}f